pkgs/tools/misc/outils/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, lib }:
+
+stdenv.mkDerivation rec {
+  pname = "outils";
+  version = "0.10";
+
+  src = fetchFromGitHub {
+    owner = "leahneukirchen";
+    repo = "${pname}";
```

```suggestion
    repo = pname;
```
`pname` is already a string.